app/src/main/res/drawable/oval_white.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="oval">
-  <solid android:color="@color/aptoide_orange"/>
+  <solid android:color="@color/default_orange_gradient_end"/>
```

Comment:
This class in v21 has color "colorPrimaryDark". Why are we using our default color here?

Reply:
It’s not possible to use ?attr in drawable for api below 21, we could create a drawable for each theme color, but given the places where the resource is being used I would just leave it as it is